R/aedseo.R

```diff
@@ -21,10 +21,14 @@
 #' Choose between "poisson," or "quasipoisson".
 #' @param na_fraction_allowed Numeric value between 0 and 1 specifying the
 #' fraction of observables in the window of size k that are allowed to be NA.
+#' @param season A numeric vector of length 2 c(start,end) with the start and
+#' end weeks of the seasons to stratify the observations by.
+#' Ex: season = c(21,20). Default is NULL.
```

Comment:
If you want to, you could render this a little nicer (in my opinion)

```suggestion
#' @param season A numeric vector of length 2 `c(start,end)` with the start and
#' end weeks of the seasons to stratify the observations by.
#' Ex: `season = c(21,20)`. Default, `NULL`, is no stratification by season.
```